migrate/20240702_load_balancer_health_probes.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_enum(:lb_node_state, %w[up down evacuating])
```

Comment:
So I don't quite understand the evacuating nomenclature, maybe I need to get used to it.

Reply:
I'm using it for VMs to be destroyed and completely removed but simply not yet because the namespace is being used by active connections.